src/ucp/core/ucp_context.c

```diff
@@ -98,6 +98,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "name, or a wildcard - '*' - which expands to all MD components.",
    ucs_offsetof(ucp_config_t, alloc_prio), UCS_CONFIG_TYPE_STRING_ARRAY},
 
+  {"SOCKADDR_TLS", "rdmacm,sockcm",
```

Comment:
`"SOCKADDR_TLS_PRIO", "rdmacm,*"`

Reply:
What is ```PRIO``` ? Just use the full name ```PRIORITY```. No need for the shortcuts in environment variables.